docker/dev/dbinit.sql

```diff
@@ -0,0 +1,7 @@
+CREATE USER backend WITH PASSWORD 'backend';
+CREATE DATABASE backend;
+ALTER DATABASE backend OWNER TO backend;
+
+CREATE USER auth WITH PASSWORD 'auth';
+CREATE DATABASE auth;
+ALTER DATABASE auth OWNER TO auth;
```

Comment:
These are both used in rafiki?
Or is the backend you are refering to there our testnet backend?

Reply:
both are for rafiki services

Reply:
got it